modules/GenericDataParser/src/server/generic_data_parser.R

```diff
@@ -2163,7 +2170,8 @@ createNewExperiment <- function(metaData, protocol, lsTransaction, pathToGeneric
     codeType = "assay",
     codeKind = "scientist",
     lsTransaction= lsTransaction)
-  experimentStatus <- racas::applicationSettings$server.sel.experimentStatus
```

Comment:
If I'm reading this correctly, the prior behavior was to set experimentStatus to the value of that `server.sel.experimentStatus` config. If that was null, then we'd fall back to "approved".

With your new optional field, we'd now take the user-supplied status from the SEL sheet if it's provided. But in the event it's not provided, I think this change removes the behavior of defaulting to `server.sel.experimentStatus` and would just _always_ default to `approved`.

I think it'd be better to have something like this, so the order of precedence is "value for SEL header > experimentStatus config > 'approved'":
```
  experimentStatus <- metaData$"Experiment Status"
  if (is.null(experimentStatus) || experimentStatus == "") {
    experimentStatus <- racas::applicationSettings$server.sel.experimentStatus
  }
  if (is.null(experimentStatus) || experimentStatus == "") {
    experimentStatus <- "approved"
  }
```

Reply:
I've implemented your recommendation.  Good catch.